wpilib/src/serial.rs

```diff
@@ -184,6 +185,42 @@ impl SerialPort {
     }
 }
 
+use std::io;
+impl io::Write for SerialPort {
+    #[inline]
+    fn write(&mut self, buf: &[u8]) -> Result<usize, io::Error> {
```

Comment:
Wouldn't using `io::Result` be clearer here?

Reply:
yep, forgot about that in my compiler-driven development